### DISCUSSION TITLE
Specifying size in pinned columns causes gaps

### DISCUSSION THREAD

**Discussion by masterbater** [KevinVandy/material-react-table#258] [Q&A]
![image](https://user-images.githubusercontent.com/26339988/207808600-dc0bad6a-d815-4937-99ca-0ab9ee82252c.png)

**Comment by KevinVandy** [comment]
Are you using virtualization or layoutMode features? I would need full code snippets or code sandboxes.

**Comment by MichalWr88** [comment]
![image](https://user-images.githubusercontent.com/22855355/234234140-5b040abc-d301-46ba-8487-2f51e093247f.png)
i have the same problem when use pinned columns and resize of columns. Pinned col have always css props right multiplier 200px  pinned col have less than 200px  and then shows gaps ( columns under pinned cols)  Is there any idea for fix it?
